src/openforms/registrations/contrib/objects_api/tests/test_backend.py

```diff
@@ -37,6 +38,29 @@ def setUpTestData(cls):
             informatieobjecttype_submission_csv="https://catalogi.nl/api/v1/informatieobjecttypen/4",
             informatieobjecttype_attachment="https://catalogi.nl/api/v1/informatieobjecttypen/3",
             organisatie_rsin="000000000",
+            content_json="""
+            {\n
+                "bron": {\n
+                    "naam": "Open Formulieren",\n
+                    "kenmerk": "{{ submission.kenmerk }}"\n
+                },\n
+                "type": "{{ productaanvraag_type }}",\n
+                "aanvraaggegevens": {% json_summary %},\n
+                "taal": "{{ submission.language_code  }}",\n
+                "betrokkenen": [\n
+                    {\n
+                    "inpBsn" : "{{ auth_bsn }}",\n
+                    "rolOmschrijvingGeneriek" : "initiator"\n
+                    }\n
+                ],\n
+                "pdf": "{{ submission.pdf_url }}",\n
+                "csv": "{{ submission.csv_url }}",\n
+                "bijlagen": [\n
+                    {% for attachment in submission.attachments %}\n
+                    "{{ attachment }}"{% if not forloop.last %},{% endif %}\n
+                    {% endfor %}\n
+                ]\n
+            }""",
```

Comment:
You can use `textwrap.dedent` to handle the indentation of the multiline string. Additionally, I don't think you need the `\n` in there, they should be included when using `"""`

Reply:
If we're providing a template here, how are we testing that our default shipped template works?